src/util-dpdk.c

```diff
@@ -66,120 +77,310 @@ void DPDKFreeDevice(LiveDevice *ldev)
 #endif
 }
 
-static FILE *HugepagesMeminfoOpen(void)
+/**
+ * Retrieves name of the port from port id
+ * Not thread-safe
+ * @param pid
+ * @return static dev_name on success
+ */
+const char *DPDKGetPortNameByPortID(uint16_t pid)
```

Comment:
This is unfortunate diffing but this function was only moved up to group it with other "global" functions